packages/engine/tests/simulations/experiment.rs

```diff
@@ -0,0 +1,269 @@
+use std::{
+    fs::{self, File},
+    io::BufReader,
+    path::{Path, PathBuf},
+    sync::Once,
+};
+
+use ::error::{ensure, Report, Result, ResultExt};
+use hash_engine::utils::OutputFormat;
+use regex::Regex;
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
+use serde_json::Value;
+
+use super::error::{ProcessContext, StatusCode, Stderr, Stdout};
+
+#[derive(Debug, Clone)]
+pub enum ExperimentType {
```

Comment:
I don't mind the duplication, especially since it looks like it keeps it simple, but can we not use the `ExperimentType` from the CLI?
`packages/engine/bin/cli/src/main.rs:59`

Reply:
`cli::main::ExperimentType` is a binary. Will use it when separating CLI into binary+library

Reply:
Ahah, good point :) Could we put a TODO?